Revert "Adds the Moodle 4.4 version of the Ilios Enrollment plugin."

lbailey-ucsf opened the pull request:
Reverts ucsf-education/moodle#89

This commit appears to have caused phpUnit test failures and is being reverted.

lbailey-ucsf committed:
Revert "Adds the Moodle 4.4 version of the Ilios Enrollment plugin."

ctam reviewed (changes requested):
Please also remove the related lines in the `.gitmodules` file to be complete.
You can also just run `git rm <path-to-submodule>`, and commit.  It used to be a lot more work to do that, but the new git version makes it easier now.
(notify @mirleu and @stopfstedt here, too, on how to properly remove a git submodule.)

ctam commented:
Hey, @lbailey-ucsf, what did you change? The tests in the GHA workflow start running again! 🥳

ctam commented:
Oh, you created the PR branch here on this repo instead of your own fork, and that's why it got run here. 🙃

lbailey-ucsf commented:
@ctam , I do not think it's possible to revert a PR from the fork side. Also, the PR in question, PR #89,  did not originate from my fork. But I'm surprised the revert didn't also revert .gitmodules because that file was as part of the commit associate with PR #89.

lbailey-ucsf committed:
Update .gitmodules to remove the enrol/ilios plugin

removed the following lines are requested by reviewer:
[submodule "enrol/ilios"]
	path = enrol/ilios
	url = https://github.com/ilios/moodle-enrol-ilios
	branch = MOODLE_404_STABLE

lbailey-ucsf commented:
@ctam did the GH_PAT expire?

ctam reviewed (approved):
lgtm
